ci/install-common-toolchain.sh

```diff
@@ -53,30 +64,22 @@ esac
     # getting HEAD is pretty safe.
     go get -v github.com/wadey/gocovmerge
 
-    echo "upgrading Pip to ${PIP_VERSION}"
```

Comment:
Installing PIP and virtualenv like this lead to some super wonky errors in Travis on Ubuntu, so I decided to just retain the pip and virtualenv's that come as part of the python we get from pyenv/brew, this all works as expected, and seems like it could end up being closer to what other folks would run.

FWIW, we don't even really use virtualenv anymore, pipenv has it vendored it and we use that to create virtualenvs for both building and testing.